lib/core/text/native.nit

```diff
@@ -29,6 +29,14 @@ redef class Byte
 			return 1
 		end
 	end
+
+	# Write self as a string into `ns` at position `pos`
+	fun add_digest_at(ns: NativeString, pos: Int) do
```

Comment:
This method has a very specific use, it should probably be private and it could be introduced in `Bytes`.